tests/models/encoder_decoder/test_modeling_tf_encoder_decoder.py

```diff
@@ -55,9 +51,7 @@
     from transformers.modeling_tf_outputs import TFBaseModelOutput
 
 if is_torch_available():
-    import torch
-
-    from transformers import BertLMHeadModel, BertModel, EncoderDecoderModel
+    pass
```

Comment:
should we just remove this `if is_torch_available():`?

Reply:
Good catch! I relied on `make fixup` to clear unused imports, and it left this pattern in a few places 👀 Removed all occurrences.